Jellyfin.Plugin.Reports/Web/reports.html

```diff
@@ -345,7 +345,7 @@
                     </select>
                 </div>
                 <button is="paper-icon-button-light" onclick="jQuery('.viewPanel').panel('toggle');" class="autoSize" tabindex="-1">
-                    <i class="md-icon">more_vert</i>
+                    <i class="material-icon more_vert"></i>
```

Comment:
I see both `material-icon` and `material-icons` in the changes.

Reply:
Good catch, is fixed